### ISSUE TITLE
[n1,n2]bdelete raises `E16: Invalid range` if buffer n1 or n2 has been wiped out

### ISSUE THREAD

**Issue by lifepillar** [vim/vim#2014]
When using `[range]bdelete`,  if one of the extremes of the range has been wiped out, then Vim gives an `E16: Invalid range`. For instance:

```vim
vim -u NONE -N file1 file2
:1bw
:1,2bdelete
```

This works as expected, however:

```vim
vim -u NONE -N file1 file2 file3
:2bw
:1,3bdelete
```

I think that the first example should work as the second one, i.e., it should ignore non-existing buffers.

**Comment by chrisbra** [comment]
That is to be expected, since 1 is [no valid buffer number](https://github.com/vim/vim/blob/master/src/ex_docmd.c#L4804) anymore. I would suggest using `%bd`, that should set the index correctly

**Comment by lifepillar** [comment]
Thanks for the info. I think that a symbol denoting “the first buffer”, in the same way as `$` denotes “the last buffer”, would make sense then, but I can live with the current behaviour. Feel free to close this.

**Comment by lacygoill** [comment]
Fixed by [8.1.1908](https://github.com/vim/vim/releases/tag/v8.1.1908).  Although, that was not the purpose of the patch.
